app/src/main/java/com/tip/lunchbox/model/zomato/RatingObj.java

```diff
@@ -0,0 +1,31 @@
+package com.tip.lunchbox.model.zomato;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class RatingObj {
+
+    @SerializedName("title")
+    @Expose
+    private Title title;
+    @SerializedName("bg_color")
+    @Expose
+    private BgColor bgColor;
+
+    public Title getTitle() {
+        return title;
+    }
+
+    public void setTitle(Title title) {
+        this.title = title;
+    }
+
+    public BgColor getBgColor() {
+        return bgColor;
+    }
+
+    public void setBgColor(BgColor bgColor) {
+        this.bgColor = bgColor;
+    }
+
+}
```

Comment:
Devansh removed these... you put it back in... why?

Reply:
We need this for the ratings.